checker/src/main/java/org/checkerframework/checker/signedness/SignednessAnnotatedTypeFactory.java

```diff
@@ -98,6 +99,8 @@ public class SignednessAnnotatedTypeFactory extends BaseAnnotatedTypeFactory {
   public SignednessAnnotatedTypeFactory(BaseTypeChecker checker) {
     super(checker);
 
+    addAliasedTypeAnnotation(NonNegative.class, SIGNEDNESS_GLB);
```

Comment:
You shouldn't need this because the Signedness Checker uses the Value Checker, which reads `@NonNegative`.  Also, I think this change is causing the test failure.